### DISCUSSION TITLE
Error due to Chinese code

### DISCUSSION THREAD

**Discussion by ShamanMap** [YaoFANGUK/video-subtitle-remover#92] [General]
你好，
非常感谢您所做的工作。
太棒了。

但是当我运行时，我收到此错误？请你帮助我好吗。

Hello,
Thank you very much for your work.
It's amazing.

But when I run, I get this error? Could you please help me.

![Warning - GitHub_ QPT-Family_QPT 1727032709](https://github.com/user-attachments/assets/e330c299-9643-4e8f-b971-16376e66ef54)

**Comment by CNkyq** [comment]
delete the space in 'SOFT SOURCE'

**Comment by ShamanMap** [comment]
wow. Xiexie Ni. Thank you very much !
